examples/common/pigweed/protos/fabric_admin_service.options

```diff
@@ -1,2 +1,3 @@
 chip.rpc.DeviceCommissioningWindowInfo.verifier max_size:97  // kSpake2p_VerifierSerialized_Length
```

Comment:
I really think this PR could help other reviewers if you let people know what they are doing. This PR is functionally doing 3 different things

1. FS accepting CCTRL command (This is plumbing data from fabric-bridge to fabric-admin, and commission the node)
2. Adding command to fabric-admin commissioning it's own fabric-bridge-app (Not needed for TE2, but will be needed in SVE to improve TH scripts)
3. Adding capability for fabric-admin to send CCTRL commands where upon receiving `ReverseOpenCommissioningWindow` fabric--admin opens up the commissioning window to it's fabric-bridge app via matter invoke commands (Not needed for TE2, but will be needed in SVE to improve TH scripts)

Reply:
After #35020 merged, this PR should be small enough to review